src/Microsoft.CodeQuality.Analyzers/CSharp/QualityGuidelines/CSharpRemoveEmptyFinalizersAnalyzer.cs

```diff
@@ -81,7 +81,7 @@ private static bool IsConditionalInvocation(InvocationExpressionSyntax invocatio
                 return false;
             }
 
-            var conditionalAttributeSymbol = WellKnownTypes.ConditionalAttribute(semanticModel.Compilation);
+            var conditionalAttributeSymbol = semanticModel.Compilation.GetTypeByMetadataName(typeof(System.Diagnostics.ConditionalAttribute).FullName);
```

Comment:
Yep, more changes I look at I don't like this. Its lot more verbose and error prone.